docs/tutorials/channels.md

```diff
@@ -513,7 +513,7 @@ fn factor_value(value: Value) -> Vec<u64> {
 
 ```
 
-Then, let's introduce _getter_ methods that are quite self-explanatory:
+Then, let's introduce _getter_ methods that are quite self-explanatory. For the validator set, we select a  rotating subset (floor((n+1)/2)) of the validators from the genesis validator set based on the height. This is done to showcase the ability to change the validator set over time (or height, in this case).
```

Comment:
Since there are multiple methods here maybe move the `get_validator_set` comment to the method comment instead and keep the comment here the way it was.